lib/cdo/script_constants.rb

```diff
@@ -110,6 +110,8 @@ module ScriptConstants
   # This can be used to override that, with lower numbers ordered sooner, and higher numbers
   # ordered later.
   CATEGORY_ORDERING_PRIORITY = {
+    # We want full courses to show up first in our list
+    full_course: -1,
```

Comment:
Are these values important (the relative positioning all that matters?)? Are these values ever persisted anywhere?

If no (yes) and no, I'd vote against moving into the negative numbers.